### PR TITLE
[Feature] Dict specs in vmas

### PR DESCRIPTION
**Pull request by matteobettini**
I had this feature in my fork for a long time and di not realize it was not in main.

This allows dictionaries as observations in VMAS

### CODE REVIEW

File: torchrl/envs/libs/vmas.py [hunk 1]
```diff
@@ -94,6 +94,17 @@ def _vmas_to_torchrl_spec_transform(
                 device=device,
             )
         )
+    elif isinstance(spec, gym_spaces.Dict):
+        spec_out = {}
+        for k in spec.keys():
+            key = k
```
Comment (vmoens) [inline torchrl/envs/libs/vmas.py:100]: what does this do?

**Review by vmoens — APPROVED** [review]
LGTM thx! Just a nit

File: torchrl/envs/libs/vmas.py (hunk 1 above)
Comment (matteobettini) [reply torchrl/envs/libs/vmas.py]: good question. I have no idea why that was in my code. I hope it was not there for some dark magic that i forgot about

**Comment by matteobettini** [comment]
let's activate the vmas tests for this

**Pull request merged**